### DISCUSSION TITLE
How to find the Speech emotion recognition overall accuracy.

### DISCUSSION THREAD

**Discussion by Lokesh97Bansal** [speechbrain/speechbrain#1827] [Q&A] [answered]
In test stats I am having test loss in train_with_wav2vec2.py file in speechbrain/receipes/IEMOCAP/emotion_recognition folder.
But I want to print the train accuracy, validation accuracy and test accuracy so that I can compare my results with the benchmark results

**Comment by BenoitWang** [comment]
Hi @Lokesh97Bansal, you can find the validation error and the test error in results/seed/train_log.txt which is `1-accuracy`, if you want to print train error you only need to do a simple modification in `on_stage_end ` method.

**Reply by Lokesh97Bansal** [reply]
Hi Thanks for the reply, I don't want the loss. I want the emotion classification accuracy percentage. I want to print the following (screen shot) accuracy. Can you please tell me how I can print this?

![image](https://user-images.githubusercontent.com/52260398/216950274-7006c0c9-dca6-439a-9a61-5db6e6f608cf.png)

**Comment by BenoitWang** [comment, marked as answer]
The loss and the classification error rate are automatically printed in `train_log.txt` that you can find in `results` folder.
![image](https://user-images.githubusercontent.com/41187612/216952137-f369ba88-baf0-4fec-bac5-612693cacc1c.png)

**Comment by Lokesh97Bansal** [comment]
Here loss is getting printed not the accuracy like 75% or so.
I want this 75% type measurement not the loss measurement.

**Reply by BenoitWang** [reply]
For example in the picture above, `test_accuracy = 1 - test_error_rate = 1 - 2.09e-01 = 7.91e-01 = 79.1%`

**Comment by Lokesh97Bansal** [comment]
I got it, thanks a lot for the help.
Also I have changed the architecture a bit. I want to print the trainable parameters for the modified architecture, but in the print it is showing the previously declared trainable parameters. Can you tell me how to change this?
Thanks.
